src/tooling/piral-cli-webpack5/webpack-hot-middleware/README.md

```diff
@@ -0,0 +1,200 @@
+# Webpack Hot Middleware
```

Comment:
I don't think we need the README, right?

Or is the plan to leave this as close as possible to the original? Also I would put it in `src/hot-middleware` instead of `webpack-hot-middleware`.